aspnetcore/blazor/forms-and-input-components.md

```diff
@@ -15,39 +15,83 @@ The Blazor framework supports forms and provides built-in input components:
 * <xref:Microsoft.AspNetCore.Components.Forms.EditForm> component bound to a model that uses [data annotations](xref:mvc/models/validation)
 * [Built-in input components](#built-in-input-components)
 
-The <xref:Microsoft.AspNetCore.Components.Forms?displayProperty=fullName> namespace provides classes for managing form views, state, and validation, and the namespace also provides access to built-in :::no-loc text="Input*"::: Razor components for use in Blazor apps. The namespace appears by default in the `_Imports.razor` file of an app created from a Blazor project template. If you don't maintain a `using` statement in `_Imports.razor` for the namespace, you can use the namespace in individual Razor components with the [`@using`](xref:mvc/views/razor#using) directive:
+The <xref:Microsoft.AspNetCore.Components.Forms?displayProperty=fullName> namespace provides classes for managing form elements, state, and validation, and the namespace also provides access to built-in :::no-loc text="Input*"::: Razor components for use in Blazor apps. The namespace appears by default in the `_Imports.razor` file of an app created from a Blazor project template. If you don't maintain a `using` statement in `_Imports.razor` for the namespace, you can use the namespace in individual Razor components with the [`@using`](xref:mvc/views/razor#using) directive:
```

Comment:
```suggestion
The <xref:Microsoft.AspNetCore.Components.Forms?displayProperty=fullName> namespace provides classes for managing form elements, state, and validation, as well as access to built-in :::no-loc text="Input*"::: components, which can be used in Blazor apps. Projects created from the Blazor project template include this namespace by default in the `_Imports.razor` file, which makes the namespace available in all the `.razor` files of the application without an explicit [`@using`](xref:mvc/views/razor#using) directive:
```

Reply:
I'm going to try again with this. The style manual doesn't favor "as well as" over "and," *and* I think we could split that into a list of two items. The shorter sentences and phrases machine translate better (so they tell us 😄).